cmd/zfs/zfs_main.c

```diff
@@ -4533,6 +4594,8 @@ zfs_do_send(int argc, char **argv)
 	if (flags.parsable && flags.verbosity == 0)
 		flags.verbosity = 1;
 
+	/* Should we warn about no -R but an exclusion list? */
```

Comment:
It does seem like this should be a fatal error.  Offhand I can't think of a case where it would make sense to specifying `-x` but not `-R`.

Reply:
Really?  Ok.  Do I need to do i18n myself, or is the call to gettext() enough? (The commit is coming in a couple of minutes.)

Reply:
I could really go either way on this, but unless somebody has a use case where this would actually make sense I'd lean towards generating an error so the user at least knows they're using it wrong.  Using `gettext()` is plenty.